lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/domain/InheritedIdentityJwtClaim.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.di.ipv.core.initialiseipvsession.domain;
+
+import java.util.List;
+
+public record InheritedIdentityJwtClaim(List<String> value) {}
```

Comment:
Sorry Mike - this has just been changed to `values` in the RFC 🤦 

(kinda makes sense, given it's a list...)

https://github.com/govuk-one-login/architecture/pull/554/files